Sources/tart/Commands/Run.swift

```diff
@@ -37,7 +37,8 @@ struct Run: AsyncParsableCommand {
       + "Note that this feature is experimental and there may be bugs present when using VNC."))
   var vncExperimental: Bool = false
 
-  @Flag var withSoftnet: Bool = false
+  @Flag(help: ArgumentHelp(visibility: .private))
+  var withSoftnet: Bool = false
```

Comment:
Maybe let's move the declaration after `netSoftnet` and add a `todo` to remove in `1.0.0`?

Reply:
How about we have an issue for that instead of a static `TODO`: https://github.com/cirruslabs/tart/issues/276?